mkdocs.yml

```diff
@@ -11,6 +11,7 @@ pages:
     - "Top bar": "using/top-bar.md"
     - "How it works": "using/how-it-works.md"
     - "Security practices": "using/security-practices.md"
+    - "Security non-events": "using/security-non-events.md"
```

Comment:
It's a little weird that this lives in `using/`. I feel like it's more `about/`. But I guess that ship has sailed since `using/` contains `how-it-works.md`. Oh well.